packages/sanity/src/desk/components/paneRouter/PaneRouterContext.tsx

```diff
@@ -23,6 +23,7 @@ export const PaneRouterContext = createContext<PaneRouterContextValue>({
   handleEditReference: () => missingContext(),
   ParameterizedLink: () => missingContext(),
   replaceCurrent: () => missingContext(),
+  closeToTheRight: () => missingContext(),
```

Comment:
**suggestion(non-blocking):**`closeToTheRight` could be renamed `closeCurrentAndAfter` or similar? (layout agnostic and a little clearer that it also closes the _current_ pane)

Alternatively, an argument could be added to `closeCurrent` to facilitate this so we don't need another method

Reply:
I agree with renaming it to `closeCurrentAndAfter`. I'd prefer a new method over adding an argument, since the former is more explicit and clear.